File: src/main/java/org/gangel/graphomance/usecases/node/CreateSingleVertexCompoundUniqueIndex.java
package org.gangel.graphomance.usecases.node;

import org.gangel.graphomance.IndexType;

public class CreateSingleVertexCompoundUniqueIndex extends CreateSingleVertexBase {

    public CreateSingleVertexCompoundUniqueIndex() {
        super("Create node with compound unique index", IndexType.UNIQUE, true, true);
    }

}

File: src/main/kotlin/org/gangel/graphomance/usecases/node/CreateSingleVertexCompoundUniqueIndex.kt
package org.gangel.graphomance.usecases.node

import org.gangel.graphomance.api.IndexType

class CreateSingleVertexCompoundUniqueIndex :
	CreateSingleVertexBase("Create node with compound unique index",
						   IndexType.UNIQUE,
						   true,
						   true) 